.github/workflows/tox.yml

```diff
@@ -0,0 +1,145 @@
+name: tox
+
+on:
+  create:  # is used for publishing to TestPyPI
+    tags:  # any tag regardless of its name, no branches
+      - "**"
+  push:  # only publishes pushes to the main branch to TestPyPI
+    branches:  # any integration branch but not tag
+      - "master"
+  pull_request:
+  release:
+    types:
+      - published  # It seems that you can publish directly without creating
+  schedule:
+    - cron: 1 0 * * *  # Run daily at 0:01 UTC
+
+jobs:
+  build:
+    name: ${{ matrix.tox_env }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tox_env: lint
+          - tox_env: docs
+          # - tox_env: py35
+          # - tox_env: py36
+          - tox_env: py37
+          # - tox_env: py38
+          # - tox_env: py39
+          - tox_env: packaging
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update \
+          && sudo apt-get install -y  ansible \
+          && ansible-doc -l | grep docker_container
```

Comment:
That is due to copy-pasta from molecule, where ansible is no longer a direct dependency and where it can run with system ansible. i will remove here to save some time and confusions.